AWSMobileAnalytics.podspec

```diff
@@ -15,4 +15,7 @@ Pod::Spec.new do |s|
   s.dependency 'AWSCore', '2.12.6'
   s.source_files = 'AWSMobileAnalytics/*.{h,m}', 'AWSMobileAnalytics/**/*.{h,m}'
   s.private_header_files = 'AWSMobileAnalytics/Internal/*.h'
+
+  s.deprecated = true
```

Comment:
Should we add this to changelog ?

Reply:
It was already deprecated, this just deprecates the Podspec, but it's probably a good idea to make an explicit note in the changelog.